docs/developer/developer-manual.html

```diff
@@ -471,7 +472,34 @@ <h2 id="Making_a_Checker_Framework_release">Making a Checker Framework release</
 <!-- This relative URL works from a clone, but not from the web. -->
 <a href="release/README-release-process.html">local version (link works from a clone)</a> or
 <a href="https://rawgit.com/typetools/checker-framework/master/docs/developer/release/README-release-process.html">web version (from previous release)</a>.
+</p>
+
 
+<h2 id="Build_historical">Building a historical version of the Checker Framework</h2>
+
+<p>
+The use of four different repositories, as explained in
+"<a href="#Related_repositories">Related repositories</a>" above, means that you
+cannot check out an old version of the Checker Framework and build it
+with <code>./gradlew assemble</code>.  By default, the Checker Framework build
+uses the latest version of the other three repositories, which are rarely
+compatible with an old version of the Checker Framework.  One symptom of the
+problem is a "Can't find stubparser jar" error message.
+</p>
+
+<p>
+To build an old version of the Checker Framework, you need to use old versions
```

Comment:
I'm not sure if this section is talking about building old *released* versions of the Checker Framework or if it's talking about building arbitrary commits in the repository.  If it's released versions, then annotation-tools should have a corresponding tag for the release.